src/test/java/com/mfarag/todo/TasksTest.java

```diff
@@ -0,0 +1,71 @@
+package com.mfarag.todo;
+
+import org.junit.Test;
+
+import java.util.Date;
+
+import static org.junit.Assert.*;
+
+public class TasksTest {
+    private Task TEST_TASK_1 = new Task ("New Task 1") ;
+    private Task TEST_TASK_2 = new Task ( "New Task 2") ;
```

Comment:
The same as the other ticket, setup the test in a `@Before` method...

This is why we try to do one pull request at a time, if something needs to be fixed, it gets fixed early and doesn't affect other code.